resolwe/flow/tests/__init__.py

```diff
@@ -199,23 +199,22 @@ def run_process(self, process_slug, input_={}, assert_status=Data.STATUS_DONE, r
         for field_schema, fields in iterate_fields(input_, p.input_schema):
             # copy referenced files to upload dir
             if field_schema['type'] == "basic:file:":
-                for app_config in apps.get_app_configs():
-
-                    old_path = os.path.join(app_config.path, 'tests', 'files', fields[field_schema['name']])
-                    if os.path.isfile(old_path):
-                        file_name = os.path.basename(fields[field_schema['name']])
-                        new_path = os.path.join(self.upload_path, file_name)
-                        shutil.copy2(old_path, new_path)
-                        self._upload_files.append(new_path)
-
-                        # since we don't know what uid/gid will be used inside Docker executor,
-                        # we must give others read and write permissions
-                        os.chmod(new_path, 0o666)
-                        fields[field_schema['name']] = {
-                            'file': file_name,
-                            'file_temp': file_name,
-                        }
-                        break
+                file_name = os.path.basename(fields[field_schema['name']])
```

Comment:
Does this work is you call tests in genialis-base? The current implementation allowed to call resolwe and resolwe_bio tests from the genialis-base project.
```
./manage.py test resolwe
```
I think you people using this Django apps has to be able to run tests on them in any Django package.